pywhat/Data/regex.json

```diff
@@ -394,6 +394,18 @@
           "Networking"
        ]
     },
+    {
+      "Name": "Amazon Standard Identification Number (ASIN)",
+      "Regex": "(?:[/dp/]|$)([A-Z0-9]{10})",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 0.5,
+      "URL": null,

```

Comment:
Hi! Thank you very much for contributing!

I have looked into this a little bit and seems there is a way to access the product page with only the ASIN number, so could you please add `https://amzn.com/` here? PyWhat will add the ASIN number at the end and users will be able to see what product it is for. 😄

Reply:
Great idea!